doc.go

```diff
@@ -21,9 +21,9 @@ The high-level bindings act like a regular net.Conn. They can be used as such. F
 	err = c.Close()
```

Comment:
At the very top, this package comment still says:

```
Package websocket provides high- and low-level bindings for the browser's WebSocket API.
```

It should be updated. How about:

```
Package websocket provides high-level bindings for the browser's WebSocket API.

These bindings offer a Dial function that returns a regular net.Conn.
They can be used similar to net package. For example:

	...
```

You can replace `if err != nil { panic(err) }` with `if err != nil { ... }` here as well, if you're okay with that.